docs/visual-basic/programming-guide/language-features/variables/object-variable-declaration.md

```diff
@@ -125,12 +125,22 @@ End Sub
   
```

Comment:
the previous lines here look odd. probably an issue with the conversion. here's what it looked like before: https://msdn.microsoft.com/en-us/library/2a5a41a3-1aa8-4236-b1f0-2382af7bf715(v=vs.120)#Flexibility of Object Variables

And double-check the hierarchy here: https://msdn.microsoft.com/en-us/library/system.windows.forms.form